### ISSUE TITLE
Tauon does not respect remapped keys in Gnome

### ISSUE THREAD

**Issue by gigawhitlocks** [Taiko2k/TauonMusicBox#665]
If the user has used the gnome-tweaks tool or the dconf-editor to customize their keyboard layout, Tauon does not respect the changes and the keys retain their original behavior inside Tauon

To reproduce:
1) Install Tauon on a machine running Gnome
2) Use gnome-tweaks to map Caps Lock to be a second control button or swap Control and Caps as seen in this tutorial https://www.emacswiki.org/emacs/MovingTheCtrlKey#h5o-3
3) Observe that the new control key does not work in Tauon and that the original control key must be pressed

**Comment by Taiko2k** [comment]
Yeah it is kind of edge case. I'm not sure how this is supposed to work or what the correct solution is. But I think reading dconf settings seems wrong  / isn't something I want to do.

**Comment by gigawhitlocks** [comment]
I get it. Feel free to close the ticket; if I happen to find time to fix it myself hopefully you'll be willing to accept a PR, and in any case I just wanted you to be aware of this behavior in case there's a more serious but related issue reported

Just trying to help out 😂
